core/src/replay_stage.rs

```diff
@@ -291,7 +291,6 @@ impl ReplayStage {
                         &bank_forks,
```

Comment:
ReplayStage needed some rejigging.  `record_rewards` needs to happen after the bank is frozen instead of immediately upon Bank creation